src/chat/__tests__/narrowsSelectors-test.js

```diff
@@ -120,11 +114,9 @@ describe('getMessagesForNarrow', () => {
       ],
     });
 
-    const anchor = getMessagesForNarrow(state, privateNarrow('john@example.com'));
-
-    const expectedState = deepFreeze([{ id: 123 }]);
```

Comment:
Much better -- thanks for cleaning these up :slightly_smiling_face: 

I think with these names you're looking at the results of two different cases of someone copy-pasting without thinking it through (or perhaps not understanding.) The `expectedState` is, as you say, a reasonable name in many reducer tests.

The `anchor`... I suspect comes from the last few test cases below, testing things like `getFirstMessageId`. There, the result is at least plausibly an "anchor" -- it's a message ID that we're going to use to identify a spot in the message stream. And then those got copied elsewhere, by a contributor who didn't understand what it meant and in a period where we weren't doing meaningful code review.

OK, and having said that I was curious to find out. :slightly_smiling_face: Indeed -- and another layer is that those tests for `getFirstMessageId` etc. were originally for a function called... `getAnchor`. a89d897c2 changed that. And before then, 8dd208120 propagated the `anchor` name to these other test cases.